### DISCUSSION TITLE
Unable to access bridge settings or to initiate search

### DISCUSSION THREAD

**Discussion by Lorccan** [maxileith/homebridge-appletv-enhanced#300] [Q&A]
Hello - I installed the plugin, but didn't have time at that point to do the pairing, so I just disabled it temporarily.  When I came back and enabled it again, I could not see Bridge Settings (so I can't set to be a child bridge) or any sign that the pairing process was starting. I've tried deleting the plugin, restarting the homebridge server and reinstalling the plugin, but I'm not getting anywhere.  (All I see in the logs relating to the plugin is: ```[01/03/2024, 16:10:27] [Homebridge UI] Running Command: /opt/homebridge/lib/node_modules/homebridge-config-ui-x/plugin-upgrade-install.sh homebridge-appletv-enhanced 1.2.5 /var/lib/homebridge/node_modules
[01/03/2024, 16:10:46] Loaded plugin: homebridge-appletv-enhanced@1.2.5
[01/03/2024, 16:10:46] Registering platform 'homebridge-appletv-enhanced.AppleTVEnhanced'```

I suspect I ruined things by not doing the pairing when I first installed the plugin, but now I am stuck!

**Comment by maxileith** [comment]
Please fill out a support request. https://github.com/maxileith/homebridge-appletv-enhanced/issues/new?template=2-support_request.yml
